installer/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/Constants.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.amazon.aws.partners.saasfactory.saasboost;
+
+import software.amazon.awssdk.core.SdkSystemSetting;
+import software.amazon.awssdk.regions.Region;
+
+public final class Constants {
+    public static final Region AWS_REGION = Region.of(System.getenv(SdkSystemSetting.AWS_REGION.environmentVariable()));
+    public static final String OS = System.getProperty("os.name").toLowerCase();
+    public static final String VERSION = Utils.version(Constants.class).split(",")[0];
```

Comment:
Yeah, I like that better. Originally I was thinking that it would provide extra information, but that information is retrievable with the commit hash anyway. I'll test to verify.